### DISCUSSION TITLE
Unable to change the display language of widgets

### DISCUSSION THREAD

**Discussion by ghost** [bluecaret/carettab#341] [Help and Support] [answered]
After I changed the language on the settings dashboard from English to Japanese, the date and clock widgets were still displayed in English instead of Japanese. How do I get the widgets to display in Japanese? Thank you.

**Comment by bluecaret** [comment, marked as answer]
You are correct, my apologies, that was an oversight. I'll work on getting that fixed. Thank you!

**Comment by bluecaret** [comment]
A fix has been made for this, an update will go out in a few days. Keep an eye out. Thank you.
